containers/solr/Dockerfile

```diff
@@ -1,14 +1,16 @@
 # syntax=docker/dockerfile:1
 FROM solr:8.11.2 as builder
 
-ENV TYPO3_SOLR=11.5.0
+ENV TYPO3_SOLR=11.5.0 \
+    SOLR_DOWNLOAD_SHA512="a0c0181993606dbaa587520e8aa8988f1b0eb845215828cf31df0dc181497ab4932db03359bc89655366197247210b4a82cbba1b7bcc5abba85545811e17eefe"
```

Comment:
Could you give this another name like `TYPO3_SOLR_DOWNLOAD_SHA512` to prevent confusion?